src/python/pants/bin/pants_loader.py

```diff
@@ -83,12 +83,7 @@ def run_alternate_entrypoint(entrypoint: str) -> None:
 
         module = importlib.import_module(module_path)
         entrypoint_fn = getattr(module, func_name)
-
-        try:
-            entrypoint_fn()
-        except TypeError:
-            print(f"{DAEMON_ENTRYPOINT} {func_name} is not callable", file=sys.stderr)
-            sys.exit(PANTS_FAILED_EXIT_CODE)
+        entrypoint_fn()
```

Comment:
This only (ever, seemingly) served to obscure unrelated `TypeErrors` occurring during engine initialization.